modal/image.py

```diff
@@ -1273,6 +1277,7 @@ def run_function(
         timeout: Optional[int] = 86400,  # Maximum execution time of the function in seconds.
         force_build: bool = False,
         secret: Optional[_Secret] = None,  # Deprecated: use `secrets`
+        **kwargs,
```

Comment:
I find it's a bit brittle to have `**kwargs` here with so many "reserved" keyword arguments to the function (that we may also extend as we add more features). I wonder if we could do something slightly different to not overload function arguments and "modal options".

The simplest solution might be to make `kwargs` (and maybe args too in that case) into a singular arguments (dict/tuple) rather than variadic ones. It's not super pretty but it's pretty uncommon and it would prevent future modal library updates from breaking people's code if we add more named args

Reply:
Agree, seems cleaner to take `kwargs: Dict[str, Any]`

Reply:
And maybe take `args: List[Any]` too?

Reply:
Good point, done.